accProcess.py

```diff
@@ -203,6 +203,26 @@ def main():
                             metavar='True/False', default=False, type=str2bool,
                             help="""Save intensity distribution
                              (default : %(default)s)""")
+    parser.add_argument('--psd',
+                            metavar='True/False', default=False, type=str2bool,
+                            help="""Calculate power spectral density for 24 hour 
+                                    circadian period
+                             (default : %(default)s)""")
+    parser.add_argument('--fourierFrequency',
+                            metavar='True/False', default=False, type=str2bool,
+                            help="""Calculate dominant frequency of sleep
```

Comment:
'of sleep for circadian rhythm analysis'

Reply:
Resolved by d43937c7eb9e1d2f76f75d6d017443c63fea9e7f